core/unit_test/cuda/TestCuda_ReducerViewSizeLimit.cpp

```diff
@@ -142,8 +148,90 @@ struct ReduceViewSizeLimitTester {
   }
 };
 
+struct TeamSizeTester {
+  void run_test_team_size(Kokkos::AUTO_t teamSizeRequested,
```

Comment:
why are you using AUTO_t here?

Reply:
Because it is one of the type of the requested values to check (Kokkos::Auto)